src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -577,6 +579,13 @@ public T Attach<T>(JsonRpcProxyOptions options)
             return proxy;
         }
 
+        public void Relay(Stream relaySendingStream, Stream relayReceivingStream, object relayTarget = null)
+        {
+            this.relayRpc = JsonRpc.Attach(relaySendingStream, relayReceivingStream, relayTarget);
```

Comment:
Setting the field even if it was set previously sounds like it would be an error. Should we throw `InvalidOperationException` when it has already been set? (tip: add a test for this).

Reply:
Yup, I haven't gone back and polished the PR.  Just wanted to get a read from you initially to see if this is the right approach.